### ISSUE TITLE
Microsoft images give errors

### ISSUE THREAD

**Issue by sicklittlemonkey** [wagoodman/dive#306]
Dive looks very cool, and today I tried it on a couple of Microsoft images, but got lots of errors. Eg:

[1mImage Source: [0mdocker://mcr.microsoft.com/dotnet/framework/aspnet
[1mFetching image...[0m (this can take a while for large images)
[1mAnalyzing image...[0m
[1mBuilding cache...[0m
  path error at layer index Index(0-1:2): unable to remove '/Files/Windows/System32/LogFiles/Scm/scm.evm': path does not exist: /Files/Windows/System32/LogFiles/Scm/scm.evm
  path error at layer index Index(0-1:2): unable to remove '/Files/Windows/System32/catroot2/edbtmp.log': path does not exist: /Files/Windows/System32/catroot2/edbtmp.log

There are 4900 lines of errors. Is this a known problem?

**Comment by thirdxchild** [comment]
Yes. I'm currently encountering the same issue. They're all "path error at layer index Index(0-14:15) unable to add..." and "path does not exist", which would be incorrect, since I'm certain that the paths reported do exist.

I've tried on several working images, which are fully functional, and the error is the same.

**Comment by timwsuqld** [comment]
I am getting these errors to for images we build using Kaniko. Maybe that's related?
```
Building cache...
  path error at layer index Index(0-34:35): unable to remove '/var/lib/apt/lists/.wh.auxfiles': path does not exist: /var/lib/apt/lists/.wh.auxfiles
  path error at layer index Index(0-35:36): unable to add '/var/lib/apt/lists/.wh.auxfiles': path does not exist: /var/lib/apt/lists/.wh.auxfiles
  path error at layer index Index(0-36:37): unable to add '/var/lib/apt/lists/.wh.auxfiles': path does not exist: /var/lib/apt/lists/.wh.auxfiles
  path error at layer index Index(0-37:38): unable to add '/var/lib/apt/lists/.wh.auxfiles': path does not exist: /var/lib/apt/lists/.wh.auxfiles
  path error at layer index Index(0-38:39): unable to add '/var/lib/apt/lists/.wh.auxfiles': path does not exist: /var/lib/apt/lists/.wh.auxfiles
...
```

**Comment by Danny5487401** [comment]
is it related to copy layer cache when i use gcr.io/kaniko-project/executor:v1.9.0-debug having same situation.
![image](https://github.com/wagoodman/dive/assets/64348131/2e07a061-a5f4-42fd-b6ef-f24247468823)

it is character device.

**Comment by IssacKramer** [comment]
+1  Same error here for windows Server Core 2022 (5G size):
" path error at layer index Index(0-47:48): unable to add ...."
